### ISSUE TITLE
Mobile menu

### ISSUE THREAD

**Issue by brianteeman** [joomla/cassiopeia#117]
### Steps to reproduce the issue
Clean installation of development branch

### Expected result
Mobile view has a mobile menu

### Actual result
menu module displayed below content

**Comment by drmenzelit** [comment]
Yes, for menus on the sidebar there is no mobile view. The problem is if a user has the main menu on the menu position (inside the header) and a sub or secondary menu on the sidebar, he/she/it will have then 2 hamburger menus?
One can change the grid order on mobile so the menu is over the content, but a sidebar can have more than a menu...

@hans2103 do you have an idea for that?

**Comment by brianteeman** [comment]
Surely whichever menu is the default should be the burger

**Comment by chmst** [comment]
The main menu is added during installation in the right sidebar. 
We can change the position in the installation sql. Then the main menu is on top and the hamburger opens the main menu. 
The main menu blog then goes to the sidebar.

I will make a pr for that. It does not afffect the cassiopeia but the joomla-cms.
